Silicon/CommonSocPkg/Include/Library/HeciLib.h

```diff
@@ -285,6 +285,76 @@ HeciGetFwCapsSkuMsg (
   OUT UINT8                      *MsgGetFwCapsAck
   );
 
+ /**
+  Send Get user caps Request
+
+  @param[out] MsgGetFwCapsAck     Return message for Get Firmware Capability SKU ACK
+
+  @exception EFI_UNSUPPORTED      Current Sec mode doesn't support this function
+  @retval EFI_SUCCESS             Command succeeded
+  @retval EFI_DEVICE_ERROR        HECI Device error, command aborts abnormally
+  @retval EFI_TIMEOUT             HECI does not return the buffer before timeout
+  @retval EFI_BUFFER_TOO_SMALL    Message Buffer is too smallfor the Acknowledge
+ **/
+EFI_STATUS
+EFIAPI
+HeciGetUserCapsSkuMsg (
+  OUT UINT8                      *MsgGetFwCapsAck
+  );
+
+
+
+  /**
+  Send Set User Capabilities State Request to ME
+
+  @param[in]   RuleData           Pointer to new rule data.
+
+  @retval EFI_UNSUPPORTED         Current ME mode doesn't support this function
+  @retval EFI_SUCCESS             Command succeeded
+  @retval EFI_DEVICE_ERROR        HECI Device error, command aborts abnormally
+  @retval EFI_TIMEOUT             HECI does not return the buffer before timeout
+  @retval EFI_BUFFER_TOO_SMALL    Message Buffer is too small for the Acknowledge
+**/
+EFI_STATUS
+HeciSetUserCapsSkuMsg (
+  IN  UINT32                      RuleData
+  );
+
+  /**
+  Send Get Firmware SKU all caps Request
+
+  @param[out] MsgGetFwCapsAck     Return message for Get Firmware Capability SKU ACK
+
+  @exception EFI_UNSUPPORTED      Current Sec mode doesn't support this function
+  @retval EFI_SUCCESS             Command succeeded
+  @retval EFI_DEVICE_ERROR        HECI Device error, command aborts abnormally
+  @retval EFI_TIMEOUT             HECI does not return the buffer before timeout
+  @retval EFI_BUFFER_TOO_SMALL    Message Buffer is too smallfor the Acknowledge
+ **/
+EFI_STATUS
+EFIAPI
+HeciGetFwAllCapsSkuMsg (
+  OUT UINT8                      *MsgGetFwCapsAck
+  );
+
+
+/**
+  Send Set FW Enabled Features Request to CSME for firmware all caps
+
+  @param[in]   RuleData           Pointer to new rule data.
+
+  @retval EFI_UNSUPPORTED         Current ME mode doesn't support this function
+  @retval EFI_SUCCESS             Command succeeded
+  @retval EFI_DEVICE_ERROR        HECI Device error, command aborts abnormally
+  @retval EFI_TIMEOUT             HECI does not return the buffer before timeout
+  @retval EFI_BUFFER_TOO_SMALL    Message Buffer is too small for the Acknowledge
+**/
+EFI_STATUS
+HeciSetFwAllCapsSkuMsg (
```

Comment:
Add EFIAPI for library function interface

Reply:
done